bin/hwinj/pycbc_generate_hwinj

```diff
@@ -141,6 +141,10 @@ parser.add_argument('--mass2', type=float, required=True,
                     help='Second mass of the binary in solar masses.')
 parser.add_argument('--inclination', type=float, required=True,
                     help='Inclination of the binary in radians.')
+parser.add_argument('--coa-phase', type=float, default=0.0,
+                    help='Reference orbital phase parameter in radians. '
```

Comment:
It does say orbital phase in the comment. And it is used to specify the gravitational wave phase in lalsimulation.  But I can be more verbose :)